libr/core/cconfig.c

```diff
@@ -3706,6 +3706,7 @@ R_API int r_core_config_init(RCore *core) {
 	SETDESC (n, "Convert string before display");
 	SETOPTIONS (n, "asciiesc", "asciidot", NULL);
 	SETBPREF ("scr.confirmquit", "false", "Confirm on quit");
+	SETBPREF ("scr.progressbar", "true", "Display a progress bar when running scripts.");
```

Comment:
i think this option should be set to false by default it can be quite anoying for batch mode. or maybe make it honor the scr.interactive variable

Reply:
I believe it does respect `scr.interactive`:

https://github.com/radareorg/radare2/blob/bd4c52f282ac3851b473a038508c55cf783af373/libr/core/cmd.c#L6852

What is the "batch mode" you're referring to?